app/Types/PostType.php

```diff
@@ -4,9 +4,9 @@
 
 class PostType extends Type
 {
-    private const TEXT = 'text';
-    private const PHOTO = 'photo';
-    private const VOTER_REG = 'voter-reg';
+    public const TEXT = 'text';
+    public const PHOTO = 'photo';
+    public const VOTER_REG = 'voter-reg';
```

Comment:
I know you were asking this and I'm not sure the answer of public vs. private here, but as it stands this file now has a mismatch of public and private!

Reply:
Yep! I wanted to make sure this is okay before changing them all in a future PR, to keep them consistent and to keep the changes down on this one.

Reply:
Got it! So I don't see any issues with these being `public` but I don't know for sure

Reply:
cc @DFurnes - I'd think this change should be okay to make -- if so, i'll open a follow up PR to make all of these types public to avoid confusion.

Reply:
These are built with [php-enum](https://github.com/myclabs/php-enum)'s `Enum` class, which I think should expose these via static methods (e.g. `PostType::VOTER_REG()` to get this enum from outside this class).

I _think_ the reason they don't suggest defining these as public is because that would allow you to change an enum at runtime by accident, whereas accessing it via the method keeps it read-only. That said, if that doesn't work I don't think it's a big deal to change!

Reply:
Although hmm, wouldn't `const` do that? If so, yeah I don't see a reason not to make public! 🤔 The only other advantage of using the static method is it returns a type-checkable instance (I think), but we don't use that here.

Reply:
The static method worked! While we're here, I'd love to eventually add a `app/Types/PostStatus` to DRY all of the hardcoded `'accepted'`, `'rejected'` status value strings with similar const / static methods.

Reply:
Yes please! 🙌

Reply:
I thought calling the static method within the factory for voter-reg posts worked, but this [unit test failed:](https://github.com/DoSomething/rogue/blob/67805b04e81c2636f171e46e701a4957e9ad2b83/tests/Unit/Models/PostModelTest.php#L197-L202)
```
Failed asserting that a row in the table [action_stats] matches the attributes {
    "action_id": 21,
    "impact": 7,
    "school_id": "101757",
    "location": "US-FL"
}.

The table is empty..

```
Something's odd with using the static method and type checking, because this function (which gets called in PostObserver) doesn't run to create action stats:

https://github.com/DoSomething/rogue/blob/67805b04e81c2636f171e46e701a4957e9ad2b83/app/Models/Post.php#L604-L617

Changed the factory back to just return `voter-reg` and the unit test passes again.